Split/Network/Streaming/SseHandler.swift

```diff
@@ -56,24 +58,57 @@ class DefaultSseHandler: SseHandler {
     private func handleOccupancy(_ notification: IncomingNotification) {
         if let jsonData = notification.jsonData {
             do {
-                let notification = try notificationParser.parseOccupancy(jsonString: jsonData)
+                let notification = try notificationParser.parseOccupancy(jsonString: jsonData,
+                                                                         timestamp: notification.timestamp)
                 notificationManagerKeeper.handleIncomingPresenceEvent(notification: notification)
             } catch {
                 Logger.w("Error while handling occupancy notification")
             }
         }
     }
 
+    private func handleControl(_ notification: IncomingNotification) {
+        if notification.timestamp <= lastControlTimestamp {
+            return
+        }
+        lastControlTimestamp = notification.timestamp
+        if let jsonData = notification.jsonData {
+            do {
+                let notification = try notificationParser.parseControl(jsonString: jsonData)
+                switch notification.controlType {
+                case .streamingPaused:
+                    broadcasterChannel.push(event: .pushSubsystemDown)
+
+                case .streamingDisabled:
+                    broadcasterChannel.push(event: .pushDisabled)
+
+                case .streamingEnabled:
+                    if notificationManagerKeeper.publishersCount > 0 {
```

Comment:
this logic must have its counterpart in NotificationKeeper, i.e., NotificationKeeper must emit `pushSubsystemUp` if and only if (1) `publishersCount` changed from 0, and (2) the last control notification was not `streamingPaused` or `streamingEnabled`.
( maybe, this can be easily resolved if both control and occupancy notifications are handled inside the NotificationManagerKeeper)

Reply:
Done!